pages/api/projects/index.js

```diff
@@ -0,0 +1,40 @@
+import mongoose from 'mongoose';
+import Middleware from '../../../middlewares';
+
+const Project = mongoose.model('Project');
+/**
+ * @async
+ * @desc Api handles project list and search request
+ * @param {{ req: NextApiRequest, res: NextApiResponse }} prop
+ */
+const IndexProjectHandler = async ({ req, res }) => {
+  if (req.method === 'GET') {
+    const limit = req.query.limit || 0;
```

Comment:
# Nitpick
A better approach would be to abstract these into a helper file or a middleware